crates/rover-config/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "rover-config"
+version = "0.1.0"
+authors = ["Ashley Williams <ashley666ashley@gmail.com>"]
```

Comment:
Can we change this to the same email we used in the rust repo (the oss email)

Reply:
oh gosh, oops, yes this pulled from my local config, great catch!